projects/packages/post-list/composer.json

```diff
@@ -27,7 +27,7 @@
 	"scripts": {
 		"build-development": [
 			"Composer\\Config::disableProcessTimeout",
-			"pnpm run start"
+			"pnpm run build"
```

Comment:
Not sure this is right, but it seems to fix the infinite build process issue when running "jetpack build packages/post-list". Probably need to look at this closer.

![Screen Capture on 2021-08-20 at 15-23-05](https://user-images.githubusercontent.com/140841/130298573-bca5acb6-960f-4e01-bff0-010c5a4d6f93.gif)

Reply:
Nah, nevermind. This worked at one point, but I'm getting this error again now. Maybe something to do with pulling from master.

```
TypeError: Cannot destructure property 'info' of 'TerserPlugin.getAsset(...)' as it is undefined.
    at TerserPlugin.taskGenerator (/Users/dustyreagan/GitHub/jetpack/node_modules/.pnpm/terser-webpack-plugin@3.1.0_webpack@4.46.0/node_modules/terser-webpack-plugin/dist/index.js:183:7)
    at taskGenerator.next (<anonymous>)
    at /Users/dustyreagan/GitHub/jetpack/node_modules/.pnpm/terser-webpack-plugin@3.1.0_webpack@4.46.0/node_modules/terser-webpack-plugin/dist/index.js:455:49
    at /Users/dustyreagan/GitHub/jetpack/node_modules/.pnpm/p-limit@3.1.0/node_modules/p-limit/index.js:23:31
    at run (/Users/dustyreagan/GitHub/jetpack/node_modules/.pnpm/p-limit@3.1.0/node_modules/p-limit/index.js:23:43)
    at /Users/dustyreagan/GitHub/jetpack/node_modules/.pnpm/p-limit@3.1.0/node_modules/p-limit/index.js:45:20
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

Reply:
Seems like if we pull from master, the error is caused by line 10 `import './style.scss';` in 'projects/packages/post-list/src/index.js'.

Reply:
This was resolved in commit 9bcc285 by updating the @wordpress/scripts devDependencies to "18.0.0" and removing "src/index.js" from the buil and start scripts in package.json.